src/main/scala/consul/v1/acl/AclRequests.scala

```diff
@@ -23,7 +23,9 @@ trait AclRequests {
 
 object AclRequests{
 
-  def apply(basePath: String)(implicit executionContext: ExecutionContext): AclRequests = new AclRequests{
+  def apply(basePath: String)(implicit executionContext: ExecutionContext, rb: ConsulRequestBasics): AclRequests = new AclRequests{
+
+    import rb._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/Codacy/scala-consul/file/1255759068/issues/source?bid=2788492&fileBranchId=2939115#l28)